pkgs/development/python-modules/rasterio/default.nix

```diff
@@ -7,14 +7,14 @@
 
 buildPythonPackage rec {
   pname = "rasterio";
-  version = "1.2.6";
+  version = "1.2.10";
 
   # Pypi doesn't ship the tests, so we fetch directly from GitHub
   src = fetchFromGitHub {
     owner = "mapbox";
     repo = "rasterio";
     rev = version;
-    sha256 = "sha256-rf2qdUhbS4Z2+mvlN1RzZvlgTgjqiBoQzry4z5QLSUc=";
+    sha256 = "sha256-xVGwQfQvxsqYihUYXENJAz9Qp9xBkhsGc/RheRTJxgo=";
   };
 
   checkInputs = [ boto3 pytest pytest-cov packaging hypothesis shapely ] ++ lib.optional (!isPy3k) mock;
```

Comment:
This should be adjusted as the module only supports [Python > 3.6](https://github.com/rasterio/rasterio/blob/master/setup.py#L321).

Please also add additionally:

```bash
  disabled = pythonOlder "3.6";
```

Could you remove the dependency on `pytest-cov`. Coverage has no benefit for us.

Reply:
Updated.